uncompyle6/semantics/pysource.py

```diff
@@ -1822,7 +1822,7 @@ def n_list(self, node):
         # will assume that if the text ends in *.
         last_was_star = self.f.getvalue().endswith("*")
 
-        if lastnodetype.endswith("UNPACK"):
+        if lastnodetype.startswith("BUILD_LIST_UNPACK"):
```

Comment:
This is more an observation rather than something necessarily something to change: BUILD_LIST_UNPACK was added in 3.5.